sensing/pointcloud_preprocessor/launch/blockage_diag.launch.xml

```diff
@@ -14,5 +15,6 @@
     <param name="vertical_bins" value="$(var vertical_bins)"/>
```

Comment:
You can remove these after those parameters are in the schema file and parameters file
"horizontal_ring_id"
"vertical_bins"
"angle_range"
"is_channel_order_top2down"